src/accelerate/tracking.py

```diff
@@ -495,8 +495,9 @@ class AimTracker(GeneralTracker):
     def __init__(self, run_name: str, logging_dir: Optional[Union[str, os.PathLike]] = ".", **kwargs):
         self.run_name = run_name
 
-        from aim import Run
+        from aim import Image, Run
 
+        self.aim_image_callable = Image
```

Comment:
Is there a reason to do `aim_image_callable` and not just do `Image`?

Reply:
I used `aim_image_callable` to remove the need for `import` statement in `log_images`. However, other trackers do `import` also in `log_images`, so I follow the current style and change it also. Thanks for pointing out that!